tools/krona.py

```diff
@@ -35,6 +35,8 @@ def import_taxonomy(self, input_tsvs, output, query_column=None, taxid_column=No
                         score_column=None, no_hits=None, no_rank=None, db=None):
         self.install()
         bin_path = os.path.dirname(self.executable_path())
+        env = os.environ.copy()
+        env['PATH'] = '{}:{}'.format(bin_path, env['PATH'])
```

Comment:
Wouldn't it make more sense to add the opt/krona/lib/ directory to env['PERL5LIB']?  This seems to be the only way to make metagenomics.krona not hit perl errors on my mac.